wasm_bindgen/raze/remote/backtrace-sys-0.1.29.BUILD

```diff
@@ -1,9 +1,21 @@
 """
```

Comment:
Does raze still generate this when you override the buildfile? Delete if not

Reply:
This is misleading because it's not an "override". It's a file that gets appended to the matching cargo-raze output file

This pointed out though that I forgot to pull a change from another branch I have. Please see  #438